docs/UserGuide.md

```diff
@@ -385,7 +386,7 @@ This command deletes an existing guest in GuestBook from the current list.
 
 The guest specified at the `INDEX` is the guest you wish to delete. 
 
-This command is typically used when a guest has checked out of the hotel. 
+This command is typically used when a guest has **checked out** of the hotel. 
```

Comment:
Should this be **check a guest out** to be consistent with add?

Reply:
Agreed!